JitsiConference.js

```diff
@@ -1351,12 +1351,26 @@ JitsiConference.prototype.onMemberJoined = function(
         JitsiConferenceEvents.USER_JOINED,
         id,
         participant);
+
+    const updateFeatures = features => {
+        participant._supportsDTMF = features.has('urn:xmpp:jingle:dtmf:0');
+        this.updateDTMFSupport();
+
+        if (features.has('http://jitsi.org/protocol/jigasi')) {
+            participant.setProperty('features_jigasi', true);
+        }
+    };
+
     this.xmpp.caps.getFeatures(jid)
-        .then(features => {
-            participant._supportsDTMF = features.has('urn:xmpp:jingle:dtmf:0');
-            this.updateDTMFSupport();
-        },
-        error => logger.warn(`Failed to discover features of ${jid}`, error));
+        .then(updateFeatures)
+        .catch(_ => { // eslint-disable-line no-unused-vars
+            // there was probably a mismatch, lets try one more time and givup
```

Comment:
Why do we do the retry?

Reply:
Cause we always hit the validation problem that Lenny recently fixed: https://github.com/jitsi/lib-jitsi-meet/commit/a89d67388f49cc18aae080cb388877380a7664b1

Reply:
But how we are hitting it since it is fixed?